docroot/modules/custom/uiowa_core/uiowa_core.module

```diff
@@ -875,3 +1003,74 @@ function uiowa_core_config_entity_filter_select(array &$form, $entity_id) {
   $form['#options'] = $options;
   unset($form['#size']);
 }
+
+/**
+ * Helper function to build card for node_view.
+ *
+ * @param array $build
+ *   A renderable array representing the entity content.
+ * @param \Drupal\Core\Entity\EntityInterface $entity
+ *   The entity object.
+ * @param array $mapping
+ *   Array of field names.
+ *
+ * @return array
+ *   The node_view renderable build array.
+ *
+ * @throws \Drupal\Core\Entity\EntityMalformedException
+ */
+function uiowa_core_node_view_build_card(array &$build, EntityInterface $entity, array $mapping): array {
+  $hide_fields = $build['#hide_fields'] ?? [];
+
+  // Map fields to the card parts.
+  foreach ($mapping as $prop => $fields) {
+    if (!is_array($fields)) {
+      $fields = [$fields];
+    }
+    if (!isset($build[$prop])) {
+      $build[$prop] = [];
+    }
+    foreach ($fields as $field_name) {
+      // @todo Refine this to remove fields if they are empty.
+      if (isset($build[$field_name]) && count(Element::children($build[$field_name])) > 0) {
+        if (!in_array($field_name, $hide_fields)) {
+          $build[$prop][$field_name] = $build[$field_name];
+        }
+        unset($build[$field_name]);
+      }
+    }
+  }
+  $link_indicator = FALSE;
+  $node_type = $entity->bundle();
+  $type_map = [
+    'page' => 'pages',
+    'person' => 'people',
+    'article' => 'articles',
+  ];
+
+  if (isset($type_map[$node_type])) {
+    $config_name = 'sitenow_' . $type_map[$node_type] . '.settings';
+    $config = \Drupal::configFactory()->getEditable($config_name);
+
+    $link_indicator = $config->get('show_teaser_link_indicator');
+  }
+  $build['#link_indicator'] = $link_indicator;
+
+  $build['#url'] = !$entity->isNew() ? $entity->toUrl('canonical')->toString() : NULL;
```

Comment:
This is related to the circle link indicator and I believe shouldn't be part of this function as it will cause it to only run if this function gets called. This should be moved back to where it originally was.